gammapy/data/metadata.py

```diff
@@ -0,0 +1,90 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from typing import Optional, Union
+from astropy.coordinates import EarthLocation, SkyCoord
+from astropy.time import Time
+from pydantic import Field, validator
+from gammapy.utils.fits import earth_location_from_dict
+from gammapy.utils.metadata import CreatorMetaData, MetaData
+
+__all__ = ["ObservationMetaData"]
+
+
+class ObservationMetaData(MetaData):
+    """Metadata containing information about the Observation.
+
+    Parameters
+    ----------
+    telescope : str, optional
+        the telescope/observatory name
+    instrument : str, optional
+        the specific instrument used
+    observation_mode : str, optional
+        the observation mode
+    location : `~astropy.coordinates.EarthLocation` or str, optional
+        the observatory location
+    deadtime : float
+        the observation deadtime. Default is 1.
+    muon_efficiency : float
```

Comment:
This quantity is really associated to HESS. Can we not add it later when loading the HESS data?
Also, for HESS, a useful quantity is TC

Reply:
I have included this for backward compatibility because there is currently a muoneff property on `Observation`. It might make more sense to have it as part of optional entries since it is quite instrument specific.

Regarding the transparency coefficient one issue is that it does not correspond to any specific keyword in gadf.